index.html

```diff
@@ -1,206 +1,405 @@
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-        
-     <button class="btn_2">See Project</button>
-     </div>
-
-     <div class="container">
-        <div> <img src="images/Portfolio_2.png" class="img_2"> </div>
-        <div class="section_1">
-         <h2 class="head_2"> Multi-Post <br> Stories</h2>
-         <ul class="list-ul">
-           <li class="list_1"> CANOPY </li>
-           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
-           <li class="list_2"><img src="images/Counter_2.png">2015</li>
-               </ul>
-         <p>A daily selection of privately 
-           personalized reads; no accounts 
-           or sign-ups required.</p>   
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-         <button class="btn_2">See Project</button>
-     </div>
-
-     <div class="container">
-        <div> <img src="images/Portfolio_3.png" class="img_2"> </div>
-        <div class="section_1">
-         <h2> Tonic </h2>
-         <ul class="list-ul">
-           <li class="list_1"> CANOPY </li>
-           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
-           <li class="list_2"><img src="images/Counter_2.png">2015</li>
-               </ul>
-         <p>A daily selection of privately
-            personalized reads; no accounts 
-            or sign-ups required.</p>   
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-         <button class="btn_2">See Project</button>
-     </div>
-
-     <div class="container">
-        <div> <img src="images/Portfolio_4.png" class="img_2"> </div>
-        <div class="section_1">
-         <h2 class="head_2"> Multi-Post <br> Stories </h2>
-         <ul class="list-ul">
-           <li class="list_1"> CANOPY </li>
-           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
-           <li class="list_2"><img src="images/Counter_2.png">2015</li>
-               </ul>
-         <p>A daily selection of privately
-            personalized reads; no accounts 
-            or sign-ups required.</p>   
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-        <button class="btn_2">See Project</button>
+  </div>
+
+  <!--section-1-->
+
+  <div class="section-2 card mobile-card">
+    <div class="card-img-div">
+      <img src="images/Portfolio_1.png">
+    </div>
+    <div class="card-details-div">
+      <h2> Tonic </h2>
+      <div class="card-highlights">
+        <p>CANOPY</p>
+        <img src="images/Counter_1.png">
+        <p> Back End Dev</p>
```

Comment:
- [ ] Kindly put this text on a single line as suggested by the previous reviewer and as presented in the Figma template. 🤓️ The issue is happening on the mobile version.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/215278183-ce2b3991-cd1c-47c7-91b9-222604d2084e.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/215278188-e9dd357a-551d-4f15-a4e8-094b32ecc7dd.png" width="300">|